examples/arm/run.sh

```diff
@@ -131,8 +139,9 @@ function run_fvp() {
         -C mps3_board.visualisation.disable-visualisation=1 \
         -C mps3_board.telnetterminal0.start_telnet=0        \
         -C mps3_board.uart0.out_file='-'                    \
+        -C mps3_board.uart0.shutdown_on_eot=1               \
         -a "${elf}"                                         \
-        --timelimit 5 || true # seconds
+        --timelimit 120 || true # seconds
```

Comment:
woah, portable q/dq are slow I guess on FVP :p

Reply:
not that slow, this is just a backstop time as the program now exits much sooner with ascii 0x4 (EOT) and the shutdown_on_eot flag above, so the timelimit is just a large one for unclean exit.

Reply:
fwiw, the mv2 emulation run is about 10 seconds on a single modern core, and exits at that point with the EOT